src/rgw/rgw_pubsub.cc

```diff
@@ -629,6 +628,25 @@ int RGWPubSub::get_topics(const DoutPrefixProvider* dpp,
   return ret;
 }
 
+int RGWPubSub::get_topics_v1(const DoutPrefixProvider* dpp,
+                             rgw_pubsub_topics& result,
+                             optional_yield y) const
+{
+  if (rgw::account::validate_id(tenant)) {
+    // if our tenant is an account, return the account listing
+    constexpr int max_items = 1000;
+    std::string start_marker, next_marker;
+    return list_account_topics(dpp, start_marker, max_items,
```

Comment:
do we support account topics in case of v1?

Reply:
I don't know, to be honest

Reply:
account users can't create topics in v1